mpas_analysis/ocean/climatology_map_waves.py

```diff
@@ -0,0 +1,526 @@
+                obs_names = ['era5', 'ss_cci']
+            else:
+                obs_names = ['era5']
+
+            for obs in obs_names:
+
+                if controlConfig is None:
+
+                    refFieldName = field[obs]
+                    diffTitleLabel = None
+                    if obs == 'era5':
+                        obs_type = 'Reanalysis'
+                        remapObservationsSubtask = \
+                            remapEraObservationsSubtask
+                        refTitleLabel = f'{obs_type} ({obs.upper()})' \
+                                        f'{era5ObsStartYear}-{era5ObsEndYear}'
+                    elif obs == 'ss_cci':
+                        obs_type = 'Observations'
+                        remapObservationsSubtask = \
+                            remapSscciObservationsSubtask
+                        refTitleLabel = f'{obs_type} ({obs.upper()})' \
+                                        f'{SscciObsStartYear}-{SscciObsEndYear}'
+
+                else:
+                    controlRunName = controlConfig.get('runs', 'mainRunName')
+                    refTitleLabel = f'{field["titleName"]}' \
+                                    f'(Control: {controlRunName})'
+                    diffTitleLabel = 'Main - Control'
+                    refFieldName = field['mpas']
+
+                outFileLabel = f'{fieldPrefix}Wave{obs}'
+
+                for comparisonGridName in comparisonGridNames:
+                    for season in seasons:
+
+                        subtaskName = f'plot{upperFieldPrefix}_{season}_' \
+                                      f'{comparisonGridName}_{obs}'
+
+                        # make a new subtask for this
+                        # season and comparison grid
+                        subtask = PlotClimatologyMapSubtask(
+                            parentTask=self,
+                            season=season,
+                            comparisonGridName=comparisonGridName,
+                            remapMpasClimatologySubtask=remapClimatologySubtask,
+                            remapObsClimatologySubtask=remapObservationsSubtask,
+                            controlConfig=controlConfig,
+                            subtaskName=subtaskName)
+
+                        subtask.set_plot_info(
+                            outFileLabel=outFileLabel,
+                            fieldNameInTitle=field['titleName'],
+                            mpasFieldName=field['prefix'],
+                            refFieldName=refFieldName,
+                            refTitleLabel=refTitleLabel,
+                            diffTitleLabel=diffTitleLabel,
+                            unitsLabel=field['units'],
+                            imageCaption=field['titleName'],
+                            galleryGroup='waves',
+                            groupSubtitle=None,
+                            groupLink='waves',
+                            galleryName=f"{field['titleName']}"
```

Comment:
There seems to be a missing space.
```suggestion
                            galleryName=f"{field['titleName']} "
```

Reply:
Yes, I had noticed this. Thanks for the reminder.